src/UKHO.SalesCatalogueStub.Api.IntegrationTests/HealthcheckTest.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+using NUnit.Framework;
+
+namespace UKHO.SalesCatalogueStub.Api.IntegrationTests
+{
+    public class HealthcheckTest
+    {
+        [Test]
+        public async Task Test_HealthcheckTest()
+        {
+            var healthcheckUrl = new Uri("https://localhost:44325/health", UriKind.Absolute);
+            var isSuccess = false;
+            var returnedValue = "";
+
+            using (var httpClient = new HttpClient())
+            {
+
+                var expectedResult = await httpClient.GetAsync(healthcheckUrl.ToString());
```

Comment:
```suggestion
                var expectedResult = await httpClient.GetAsync(healthcheckUrl);
```
No need for a string convert as there is a `GetAsync(Uri)` overload.